ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -124,9 +124,9 @@ export default class StatusDropdown extends React.Component {
       'ddd MMMM D YYYY',
     );
 
-    const bugTitle = `${getTitle(alertSummary)} regression on push ${
-      alertSummary.revision
-    } (${pushDate})`;
+    const bugTitle = `${getFilledBugSummary(
+      alertSummary,
+    )} regression on ${pushDate}`;
```

Comment:
We could include the "regression on ..." into `getFilledBugSummary()` as well. `alertSummary` param has all the required context.